Src/FluentAssertions/Formatting/FormattedObjectGraph.cs

```diff
@@ -128,5 +140,12 @@ public override string ToString()
         return string.Join(Environment.NewLine, lines.Concat(new[] { lineBuilder.ToString() }));
     }
 
-    private string Whitespace => new(' ', indentation * SpacesPerIndentation);
+    internal PossibleMultilineFragment StartPossibleMultilineFragment()
```

Comment:
🤔 You're not _creating_ a multi-line fragment. What you are trying to do is to keep all successive formatting attempts on a single line, until one requires a new line. Only then do you switch to a multi-line formatted graph. Also, you're "starting" something, but we're never explicitly "stopping" this. So maybe a `using` construct would be better.

I'm still thinking of a better name, but I'm not sure yet. Will get back to you on that.

Reply:
`AsSingleLineUntilNoLongerPossible`?
`KeepOnSingleLineAsLongAsPossible`?
`GetSingleLineFormatter`?

Reply:
I quite like `KeepOnSingleLineAsLongAsPossible`?